dist/main/lang/fixmyts/quickFixes/addClassMethod.js

```diff
@@ -49,7 +49,7 @@ var AddClassMethod = (function () {
             typeString = getTypeStringForNode(binaryExpression.right, info.typeChecker);
         }
         else if (parentOfParent.kind == 165) {
-            var nativeTypes = ['string', 'number', 'boolean', 'object'];
+            var nativeTypes = ['string', 'number', 'boolean', 'object', 'null', 'undefined', 'RegExp'];
```

Comment:
Weird, why does it show this line as changed? Seems like it's exactly the same here in the master branch: https://github.com/TypeStrong/atom-typescript/blob/master/dist/main/lang/fixmyts/quickFixes/addClassMethod.js#L52